pkg/kubernetes/diff.go

```diff
@@ -209,3 +217,13 @@ func (m multiDiff) diff() (*string, error) {
 	}
 	return &diff, nil
 }
+
+func findNamespaces(state manifest.List) map[string]bool {
```

Comment:
This behavior is generally useful enough to put it onto `manifest.List` directly, maybe like so?

```
func (l manifest.List) Namespaces() []string

Reply:
sure!

Reply:
done